plane_spotter/geolocator.py

```diff
@@ -0,0 +1,13 @@
+class Geolocator:
+  ''' Translates coordinates to the closest airport. '''
+
+  def __init__(self, airport_code_file='plane_spotter/data/airport-codes.csv'):
+    ''' Initializes a Geolocator object '''
+    pass
+
+  def lookup_airport(self, coordinates, max_distance):
```

Comment:
I like giving type annotations like this:

```python
  def lookup_airport(self, coordinates: tuple[float, float], max_distance: float):
```

Otherwise looks good!

Reply:
No problem! I always forget about typing hints.